GOVERNANCE.md

```diff
@@ -0,0 +1,67 @@
+# Kubeapps Governance
```

Comment:
I was looking for a link to something on which this governance doc is based? Is there one? It's quite interesting, and worth indicating the source if so, imo.

Reply:
Added some credits. I saw other oss projects in Github doing so.